content/post/2019-11-05.Rmd

```diff
@@ -0,0 +1,802 @@
+---
+title: "Spatial Data Analysis with INLA"
+author: "Virgilio Gómez Rubio"
+date : 2019-11-05
+categories: ["R"]
+tags: ["R", "INLA", "spatial"]
+---
+
+## Introduction
+
+In this session I will focus on Bayesian inference using the integrated
+nested Laplace approximation (INLA) method. This is described in Rue et al.
```

Comment:
La frase de la cita Rue et al. me suena rara. ¿No sería?:
**As described** in Rue et al. 2019, **INLA** can be used... This **method** is implemented